src/infrastructure/figma/figma-service.test.ts

```diff
@@ -176,36 +178,37 @@ describe('FigmaService', () => {
 			});
 		});
 
-		it('should throw when a request to a figma api fails', async () => {
```

Comment:
Is it valuable to keep the "figma api fails" tests as well?

Reply:
Taking into account, that the method does error translation, yes! Returned the test!